pom.xml

```diff
@@ -1755,7 +1755,7 @@
         <carbon.multitenancy.version>4.7.8</carbon.multitenancy.version>
         <!-- For carbon based products use 0.12.x jaggery version. 0.13.x is not for carbon based products
```

Comment:
This comment says not to update jaggery. But we are updating here. Can we change the comment then?

Reply:
I will remove it.